react/features/shared-video/middleware.any.js

```diff
@@ -36,6 +37,25 @@ MiddlewareRegistry.register(store => next => action => {
     const { ownerId: stateOwnerId, videoUrl: statevideoUrl } = state['features/shared-video'];
 
     switch (action.type) {
+    case CONFERENCE_JOIN_IN_PROGRESS: {
+        conference.addCommandListener(SHARED_VIDEO,
```

Comment:
this variable is set from the state, maybe take it from the action? also, getting all those state variables outside of the `case` seems wrong, since it will be done for *any* action! Can you move them inside the case pl?

Reply:
Actually, this is super weird. We should be getting those inside the handler, where they are used!